README.md

```diff
@@ -56,13 +56,13 @@ providers:
     zone: europe-west1-b
 ```
 
-Please note also that most of the variables defined there can be superseeded on the command line by uppercasing them and prefixing by HSSH_. For instance:
+Please note also that most of the variables defined there can be superseeded on the command-line by uppercasing them and prefixing by HSSH_. For instance:
 
 ```
-HSSH_DEBUG=1 AWS_PROFILE=dev HSSH_INTERACTIVE=1 hssh -o "StrictHostKeyChecking no" -o "UserKnownHostsFile /dev/null" core@nomad
+HSSH__DEBUG=1 AWS_PROFILE=dev HSSH_INTERACTIVE=1 hssh -o "StrictHostKeyChecking no" -o "UserKnownHostsFile /dev/null" core@nomad
```

Comment:
single underscore not double !!!

Reply:
My bad. My vim replace `HSSH_DEBUG ` into `HSSHDEBUG` 🤦‍♂️